core/papersize.lua

```diff
@@ -63,21 +63,26 @@ local papersize = {
 }
 
 setmetatable(papersize, {
-    __call = function (self, size)
-      local _, _, x, y = string.find(size, "(.+)%s+x%s+(.+)")
-      if x and y then
-        return { SILE.measurement(x):tonumber(), SILE.measurement(y):tonumber() }
-      else
-        size = string.lower(size:gsub("[-%s]+", ""))
-        if self[size] then
+  __call = function(self, size, orientation)
+    local _, _, x, y = string.find(size, "(.+)%s+x%s+(.+)")
+    if x and y then
+      return { SILE.measurement(x):tonumber(), SILE.measurement(y):tonumber() }
+    else
+      size = string.lower(size:gsub("[-%s]+", ""))
+      if self[size] then
+        if orientation == "landscape" then
```

Comment:
Could be good to check for "portrait" and error if orientation is not valid.

Reply:
(Or possibly change it to a boolean `landscape=true/false`, are there any other orientation than landscape/portrait?)

Reply:
What about 180/270° rotations? I could see those being more likely to be used per-page than per-document, but it would be nice to know if there is any precedent out there for any other orientations. If not we either need to validate the two enum options or make it a boolean as mentioned.

Reply:
These other rotations are interesting but given its use being per-page, they could be implemented as commands instead of being passed as options to the class.

Reply:
> What about 180/270° rotations?

That's possibly something different. 
-There are the paper dimensions (e.g. one does not really need rotating anything for making slides with SILE, but just inverted width x height).
- Rotating page/frames (possibly per-page) is something perhaps different, whether done in a similar way as https://github.com/PietroCarrara/sine did for their booklets, or in other ways (frame specification, outputter logic, etc.)

Reply:
I think this is correct, other rotations are a kind of different thing.

Here is my current thought. We can go ahead and implement a landscape/portrait toggle. I think this should probably be a boolean where papersize is the portrait orientation and `landscape=true` just flip-flops the X and Y dimensions. Nothing else would change about direction/etc. If in the future we have a more generic rotation/orientation thing we can figure out what the interplay is, and probably the handling order will be papersize, adjust papersize based on landscape toggle, then rotate or otherwise orientate things.